1-0-rdbms-and-sql/1-2-1-one-to-one-schema/src/test/java/com/bobocode/UserProfileDbInitializerTest.java

```diff
@@ -0,0 +1,347 @@
+        }
+    }
+
+    @Test
+    @Order(12)
+    @DisplayName("profiles table group id is Bigint")
+    void testProfilesGroupIdTypeIsBigint() throws SQLException {
+        try (Connection connection = dataSource.getConnection()) {
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery("SELECT * FROM INFORMATION_SCHEMA.COLUMNS" +
+                    " WHERE table_name = 'profiles' AND column_name = 'user_id';");
+
+            resultSet.next();
+            String idTypeName = resultSet.getString("type_name");
+
+            assertThat(idTypeName).isEqualTo("BIGINT");
+        }
+    }
+
+    @Test
+    @Order(13)
+    @DisplayName("profiles table has all the required columns")
+    void testProfilesTableHasAllRequiredColumns() throws SQLException {
+        try (Connection connection = dataSource.getConnection()) {
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery("SELECT * FROM INFORMATION_SCHEMA.COLUMNS" +
+                    " WHERE table_name = 'profiles';");
+
+            List<String> columns = fetchColumnValues(resultSet, "column_name");
+
+            assertThat(columns.size()).isEqualTo(5);
+            assertThat(columns).containsExactlyInAnyOrder("user_id", "job_position", "company", "education", "city");
+        }
+    }
+
+    @Test
+    @Order(14)
+    @DisplayName("profiles table string columns have correct type and length")
+    void profilesTableStringColumnsHaveCorrectTypeAndLength() throws SQLException {
+        try (Connection connection = dataSource.getConnection()) {
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery("SELECT column_name FROM INFORMATION_SCHEMA.COLUMNS" +
+                    " WHERE table_name = 'profiles' AND type_name = 'VARCHAR' AND character_maximum_length = 255;");
+
+            List<String> stringColumns = fetchColumnValues(resultSet, "column_name");
+
+            assertThat(stringColumns.size()).isEqualTo(4);
+            assertThat(stringColumns).containsExactlyInAnyOrder("job_position", "company", "education", "city");
+        }
+    }
+
+    @Test
+    @Order(15)
+    @DisplayName("profiles table has primary key")
+    void profilesTablesHasPrimaryKey() throws SQLException {
+        try (Connection connection = dataSource.getConnection()) {
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery("SELECT * FROM INFORMATION_SCHEMA.CONSTRAINTS" +
+                    " WHERE table_name = 'profiles' AND constraint_type = 'PRIMARY_KEY';");
+
+            boolean resultIsNotEmpty = resultSet.next();
+
+            assertThat(resultIsNotEmpty).isEqualTo(true);
+        }
+    }
+
+    @Test
+    @Order(16)
+    @DisplayName("profiles table primary key has correct name")
```

Comment:
@mathodcoast let's start all tests names from the capital letter, in this case we case say 
`The profiles table primary key constraint name is "profiles_PK"`